### ISSUE TITLE
Long room address overflows general room setting tab

### ISSUE THREAD

**Issue by luixxiul** [element-hq/element-web#25614]
### Steps to reproduce

1. Create a room
2. Set the room address to something like `Loremipsumdolorsitametconsecteturadipisicingelitseddoeiusmodtemporincididuntutlaboreetdoloremagnaaliqdoloremagnaaliqdoloremagnaaliq:matrix.org`

#### What happened instead?

![0](https://github.com/vector-im/element-web/assets/3362943/cf4456a1-e299-4b3e-a6a3-fb85b5ab6ad9)

### Operating system

Debian

### Browser information

Firefox

### URL for webapp

app.element.io

### Application version

1.11.33

### Will you send logs?

No

**Comment by luixxiul** [comment]
Create a E2E test to confirm the fix along with the fix itself.

**Comment by Rohit-Singh-Rawat** [comment]
Can u assign me this issue .

**Comment by t3chguy** [comment]
Thank you for your interest in this issue. We will normally not assign the issue to an external contributor until they have provided at least a draft PR which is taking the right direction. For further details, see our [guidance on assigning issues](https://github.com/vector-im/element-meta/wiki/Guidance-on-assigning-issues).

If you have any technical questions about this issue, you can ask for help in [#element-dev:matrix.org](https://matrix.to/#/#element-dev:matrix.org)

**Comment by NEUDitao** [comment]
So would the fix be moreso "prevent this from happening," or "allow useres to input these inputs, but let's make it so the text box actually shows the end?"

I don't know if there's a designer behind the app/a design philosophy, so I don't want to step on any toes here if I put up a draft

**Comment by t3chguy** [comment]
@NEUDitao you can't prevent this from happening given other clients may cause it and it may already be the set room address for rooms. Its just about handling it more sanely, ellipsis etc

**Comment by amanopia** [comment]
Hey! I was trying to reproduce this issue on my system by setting up the application. But I am having trouble with starting the application and instead getting this error.

<img width="646" alt="image" src="https://github.com/vector-im/element-web/assets/60280815/85537c5e-fc8d-4763-b2ec-f0d36cfde804">
And when I am trying to locate this file, nothing is turning up. 
 I have installed all the dependencies for  **matrix-js-sdk** and **matrix-react-sdk** as well as for the main project. But when I am checking in the github desktop application, I can see that the version of dependencies has been updated. Could that be a problem? Also, I accidentally ran **npm install** the first time while I was setting up the application instead of using **yarn install**. Could that be a potential cause? Would cloning the whole thing again and redoing the steps help? Also I would really appreciate it if someone can explain why do we need to copy the config.sample.json file to config.json? I am suspecting might be so that we can customize it according to our needs.
I am new to open source, can someone help me with the issue? Any help would be appreciated. Thanks!
